bin/ember-template-lint.js

```diff
@@ -33,7 +33,7 @@ async function buildLinterOptions(workingDir, filePath, filename = '', isReading
     let resolvedFilePath = path.resolve(workingDir, filePath);
     let source = await readFile(resolvedFilePath, { encoding: 'utf8' });
 
-    return { source, filePath, moduleId };
+    return { source, filePath, moduleId, workingDir };
```

Comment:
We shouldn't add this here, the linter already has the `workingDir`:

https://github.com/ember-template-lint/ember-template-lint/blob/4be6bf9f0421cca14e8ac9d10fc154cdeeb1c7c1/bin/ember-template-lint.js#L239-L245